src/main/java/com/denizenscript/ddiscordbot/commands/DiscordInteractionCommand.java

```diff
@@ -87,57 +89,27 @@ public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException
                     && arg.matchesEnum(DiscordInteractionInstruction.values())) {
                 scriptEntry.addObject("instruction", arg.asElement());
             }
-            else if (!scriptEntry.hasObject("interaction")
-                    && arg.matchesPrefix("interaction")
-                    && arg.matchesArgumentType(DiscordInteractionTag.class)) {
-                scriptEntry.addObject("interaction", arg.asType(DiscordInteractionTag.class));
-            }
-            else if (!scriptEntry.hasObject("ephemeral")
-                    && arg.matchesPrefix("ephemeral")) {
-                scriptEntry.addObject("ephemeral", new ElementTag(arg.getValue()));
-            }
-            else if (!scriptEntry.hasObject("attach_file_name")
-                    && arg.matchesPrefix("attach_file_name")) {
-                scriptEntry.addObject("attach_file_name", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("attach_file_text")
-                    && arg.matchesPrefix("attach_file_text")) {
-                scriptEntry.addObject("attach_file_text", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("rows")
-                    && arg.matchesPrefix("rows")) {
-                scriptEntry.addObject("rows", arg.object);
-            }
-            else if (!scriptEntry.hasObject("message")) {
+            else if (!scriptEntry.hasObject("message")
+                    && !arg.hasPrefix()) {
                 scriptEntry.addObject("message", new ElementTag(arg.getRawValue()));
             }
-            else {
-                arg.reportUnhandled();
```

Comment:
(per discord discussion) this shouldn't be removed, also arguments should be registered